coalahtml/_coalahtml/index.html

```diff
@@ -9,9 +9,9 @@
     <link rel='stylesheet' type='text/css'
```

Comment:
@AbdealiJK  Hm right. Is there a way we can have this as stand alone python package and still have a demo? Is it possible to change the gh-page entry point?